ci/workflows/main-cron.yml

```diff
@@ -307,4 +307,4 @@ steps:
     # - Nexmark event generation: ~3min for 100mil records.
     # - Generate Flamegraph: ~15min (see https://github.com/koute/not-perf/issues/30 on optimizing)
     # - Building RW artifacts: ~8min
-    timeout_in_minutes: 50
\ No newline at end of file
+    timeout_in_minutes: 120
```

Comment:
Is this enough for all queries? 👀 What about triggering this manually to have a test?

Reply:
Should be, currently I have one running on `main-cron`: https://buildkite.com/risingwavelabs/main-cron/builds/455#0187c0e5-10c2-42e2-a2b2-86df8484c48a

Reply:
Will update on the results

Reply:
Sorry I'm wrong, maybe 5 hours is a safer bet.

Reply:
```bash
> ls ../scripts/sql/nexmark/ | grep "q[0-9]*\.sql" | wc -l | sed 's/\(.*\)/\1 \* 15 \/ 60/' | bc
5
```

Assuming worst case of ~15min per query, that would be 5hrs.

Reply:
Trigger a new build: https://buildkite.com/risingwavelabs/main-cron/builds/456

Reply:
Passed in 4hrs, q17 flamegraph looks okay now.

all 23 flamegraphs can be found here: https://buildkite.com/risingwavelabs/cpu-flamegraph/builds/11#0187c1c5-1ac0-4da5-8f6a-4e61a804f832